website/templates/project/contributors.mako

```diff
@@ -57,6 +58,11 @@
                   <i class="fa fa-plus"></i> Add
                 </a>
             <!-- /ko -->
+            % if node['mailing_list_enabled'] and not node['is_registration']:
```

Comment:
Why do we need `node['is_registration']` here? If it's possible to have mailing lists enabled on a registration, then why don't we have the `not node['is_registration']` constraint on lines 243 and 327 as well? If it's not possible, then we do we have it on this line?